src/components/Install/Install.test.tsx

```diff
@@ -1,11 +1,48 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
+
+import { DetailContext, DetailContextProps } from '../../pages/Version';
+import data from '../Versions/__partials__/data.json';
```

Comment:
Better duplicate than reference other component resources, if someone needs to change the metadata or make it smaller will break your test. The better here is create your own `__partials__`, isolated test resources helps with maintenance in long term.